src/main/resources/templates/partials/forms.html

```diff
@@ -95,7 +95,7 @@ <h2 class="govuk-fieldset__heading" th:text="${formLabel}"></h2>
          th:id="${fieldId}"
          th:name="${fieldId}"
          type="text"
-         th:attr="aria-label=${hideLabel ? label : null}"
+         th:attr="aria-label=${hideLabel ? label : null}, maxlength=${maxlength != null ? maxlength : null}"
```

Comment:
I think this can be simplified

```suggestion
         th:attr="aria-label=${hideLabel ? label : null}, maxlength=${maxlength}"
```